spec/requests/v1/submissions_spec.rb

```diff
@@ -177,4 +177,14 @@
       expect(json.dig('data', 'attributes', 'data', 'test')).to eql 'test'
     end
   end
+
+  describe 'GET /submissions/:submission_id/calculate' do
+    it 'invokes the calculate lambda function successfully' do
+      submission = FactoryBot.create(:submission)
+
+      get "/v1/submissions/#{submission.id}/calculate"
```

Comment:
This should be a `POST` - `GET` methods should be non-destructive, whereas this causes something to happen.